python_modules/dagster/dagster_tests/definitions_tests/module_loader_tests/test_module_loaders.py

```diff
@@ -253,6 +254,24 @@ def test_load_from_definitions_from_current_module(
         assert len(obj_ids) == len(expected_obj_ids)
 
 
+@pytest.mark.parametrize(**ModuleScopeTestSpec.as_parametrize_kwargs(MODULE_TEST_SPECS))
+@patch("dagster._core.definitions.module_loaders.load_defs_from_module.find_modules_in_package")
```

Comment:
[1] mock `find_modules_in_package` now that it is tested on its own in `test_utils.py`

Reply:
Nice. Agree that it should be tested on its own and that this simplifies the testing strategy